lib/xcode.js

```diff
@@ -198,6 +199,36 @@ const getMaxIOSSDK = _.memoize(
   }
 );
 
+function searchForConnectedDevices(obj, acc) {
```

Comment:
why not just use an extra loop? seems like we only ever call this function externally with an array arg